charts/argo-cd/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v2.1.7
 description: A Helm chart for ArgoCD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 3.27.1
+version: 3.27.2
```

Bump minor (as it adds a feature)
```suggestion
version: 3.28.0
```